subprojects/packagefiles/lz4/meson.build

```diff
@@ -383,4 +383,15 @@ project(
   meson_version: '>=0.49.0'
 )
 
+if host_machine.system() == 'windows' and get_option('default_library') != 'static'
```

Comment:
Probably worth adding a comment about why this is here?

Reply:
Added.